canonical-web-and-design/snapcraft.io
# Add noscript pixel

## Done
Added a small image to the page in a `noscript` tag

## QA
Disable JS and check that the `pixel.png` is downloaded

- `dotrun`
- http://0.0.0.0:8004/_status/metrics
```
users_with_js_total 0.0
users_without_js_total 0.0
```
- Those values should be 0
- Navigate on the site
- The `users_with_js_total` should be increased
- The `users_without_js_total` should 0
- Disable js
- Navigate on the site
- The `users_with_js_total` should be increased
- The `users_without_js_total` should be increased
